torch/_inductor/codegen/pallas.py

```diff
@@ -187,63 +187,184 @@ def minimum(a: str, b: str) -> str:
     def where(cond: str, a: str, b: str) -> str:
         return f"jnp.where({cond}, {a}, {b})"
 
+    @staticmethod
+    def to_dtype(
+        x: str,
+        dtype: torch.dtype,
+        src_dtype: Optional[torch.dtype] = None,
+        use_compute_types: bool = True,
+    ) -> str:
+        # Map PyTorch dtype to JAX dtype
+        dtype_map = {
+            torch.float32: "jnp.float32",
+            torch.float64: "jnp.float64",
+            torch.float16: "jnp.float16",
+            torch.bfloat16: "jnp.bfloat16",
+            torch.int32: "jnp.int32",
+            torch.int64: "jnp.int64",
+            torch.int16: "jnp.int16",
+            torch.int8: "jnp.int8",
+            torch.uint8: "jnp.uint8",
+            torch.bool: "jnp.bool_",
+        }
+        jax_dtype = dtype_map.get(dtype, f"jnp.{dtype}")
```

Comment:
Create a helper to map torch types to jnp.*.

I think you had a similar dict in the prior PR (in the output code), let's combine this into one place.